src/bindings/python/src/openvino/frontend/pytorch/torchdynamo/op_support.py

```diff
@@ -250,6 +254,7 @@ def __init__(self, options):
             "torch.ops.aten.unsqueeze.default": None,
             "torch.ops.aten.unsqueeze_copy.default": None,
             "torch.ops.aten.upsample_nearest2d.default": None,
+            "torch.ops.aten.upsample_nearest3d.vec": None,
```

Comment:
Could you please support torch.ops.aten.upsample_nearest2d.vec as well? Yolo12 does not work without it

```suggestion
            "torch.ops.aten.upsample_nearest2d.default": None,
            "torch.ops.aten.upsample_nearest2d.vec": None,
            "torch.ops.aten.upsample_nearest3d.vec": None,
```